src/js/services/forms.js

```diff
@@ -97,15 +97,15 @@ export default App.extend({
       });
     });
   },
-  fetchLatestFormSubmission() {
+  fetchLatestFormSubmission(flowId) {
```

Comment:
Looking at this now, should this be `(flowId = null)`. But I guess it's fine if `flowId` is just left as `undefined`...

Reply:
undefined it better in this case I think.  null is `&flow=` and undefined is  ``